translate/filters/checks.py

```diff
@@ -2312,6 +2312,28 @@ def __init__(self, **kwargs):
         StandardChecker.__init__(self, **kwargs)
 
 
+minimalconfig = CheckerConfig()
+
+
+class MinimalChecker(StandardChecker):
+
+    def __init__(self, **kwargs):
+        checkerconfig = kwargs.get("checkerconfig", None)
+
+        if checkerconfig is None:
+            checkerconfig = CheckerConfig()
+            kwargs["checkerconfig"] = checkerconfig
+
+        limitfilters = kwargs.get("limitfilters", None)
+
+        if limitfilters is None:
+            limitfilters = ["untranslated", "unchanged", "blank", "doublewords"]
```

Comment:
I'd leave doublewords out of this one as that's more stylistic and the other ones really are teasing out missing translations.